storage-server-api/src/main/java/org/jetbrains/ruby/codeInsight/types/storage/server/impl/Schema.kt

```diff
@@ -15,27 +15,28 @@ object GemInfoTable : IntIdTable() {
     val version = varchar("version", GemInfo.LENGTH_OF_GEMVERSION)
 }
 
-class GemInfoData(id: EntityID<Int>) : IntEntity(id), GemInfo {
-    companion object : IntEntityClass<GemInfoData>(GemInfoTable)
+class GemInfoRow(id: EntityID<Int>) : IntEntity(id), GemInfo {
```

Comment:
Could you please elaborate on such renames? What sense does the "Row" word carry here?

By the way, when I was working on the project, it turned out that DAO brings huge overhead so actually it would be nice to get rid of it, especially for more used entities, like classes/methods and signatures. I've created #27 for that.

Reply:
Well, looking at Exposed readme, the different naming is proposed (I mean there is no "Row" there). I am OK with removing `Data`, though, after removing DAO ofc.